protoc-gen-grpc-gateway/descriptor/types.go

```diff
@@ -453,14 +453,14 @@ var (
 	wellKnownTypeConv = map[string]string{
 		".google.protobuf.Timestamp":   "runtime.Timestamp",
 		".google.protobuf.Duration":    "runtime.Duration",
-		".google.protobuf.StringValue": "runtime.StringValue",
-		".google.protobuf.BytesValue":  "runtime.BytesValue",
-		".google.protobuf.Int32Value":  "runtime.Int32Value",
-		".google.protobuf.UInt32Value": "runtime.UInt32Value",
-		".google.protobuf.Int64Value":  "runtime.Int64Value",
-		".google.protobuf.UInt64Value": "runtime.UInt64Value",
-		".google.protobuf.FloatValue":  "runtime.FloatValue",
-		".google.protobuf.DoubleValue": "runtime.DoubleValue",
-		".google.protobuf.BoolValue":   "runtime.BoolValue",
+		".google.protobuf.StringValue": "runtime.String",
```

Comment:
@johanbrandhorst seems it's still broken.

I try doing `go install` where my `*.pb.gw.go` files are.
Throws: `cannot assign string to protoReq.Domain (type *wrappers.StringValue) in multiple assignment`

Reply:
One way I could find is to update our `runtime/convert.go` to support `StringValue`.
```                                                                                                                                       
func StringValue(val string) (*wrappers.StringValue, error) {                                                                                
        return &wrappers.StringValue{Value: val}, nil                                                                                        
}
```

and keep the map here in `types.go` as:
```
".google.protobuf.BoolValue":   "runtime.BoolValue"
```